src/VisualStudio/CSharp/Impl/Options/AutomationObject.cs

```diff
@@ -515,16 +515,16 @@ public int Space_WithinSquares
             set { SetBooleanOption(CSharpFormattingOptions.SpaceWithinSquareBrackets, value); }
         }
 
-        public int Style_PreferIntrinsicPredefinedTypeKeywordInDeclaration
+        public string Style_PreferIntrinsicPredefinedTypeKeywordInDeclaration
```

Comment:
This place is just for our options serialization. I don't think it will be a breaking change.